php_phongo.c

```diff
@@ -234,8 +234,27 @@ static void php_phongo_log(mongoc_log_level_t log_level, const char *log_domain,
 
 /* }}} */
 
+/* Splits a namespace name into the database and collection names, allocated with estrdup. */
+static bool phongo_split_namespace(const char *namespace, char **dbname, char **cname) /* {{{ */
```

Comment:
I'd have preferred the forwards declaration as it would minimize changes that are not really related to the change at hand.